src/modules/contacts/pages/ContactOverviewPage.vue

```diff
@@ -108,6 +108,12 @@
           >{{ t('form.saveChanges') }}</AppButton
         >
       </form>
+      <AppAlert v-if="hasSetAnnotations" class="mt-5">
```

Comment:
I think for consistency with the other inline success boxes this should be a `<MessageBox type="success">`

Reply:
Thanks, did that